### PR TITLE
docs: Fix wavefront config file docs for using proxy server.

### PR DESCRIPTION
**Pull request by keep94**
### Required for all PRs:

- [X ] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [ X] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

Fixed README file so that it is clear to user to use url in config file for direct ingestion and to use host and port for proxy server.

### CODE REVIEW

**Comment by keep94** [comment]
!signed-cla

File: plugins/outputs/wavefront/README.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ This plugin writes to a [Wavefront](https://www.wavefront.com) proxy, in Wavefro
 ## Configuration
 
 ```toml
-  ## Url for Wavefront Direct Ingestion or using HTTP with Wavefront Proxy
-  ## If using Wavefront Proxy, also specify port. example: http://proxyserver:2878
+  ## Url for Wavefront Direct Ingestion.
```
Comment (powersj) [inline plugins/outputs/wavefront/README.md:8]: Is a port not required for a proxy?

Comment (keep94) [reply plugins/outputs/wavefront/README.md:8]: Port is required for a proxy, see below, but the port is not required for direct ingestion.

Comment (powersj) [reply plugins/outputs/wavefront/README.md:8]: ok, but why are you removing this from the docs?

Comment (oppegard) [reply plugins/outputs/wavefront/README.md:8]: @powersj we had a customer run into an issues using Telegraf with Wavefront Proxy – when we asked them to switch to the Proxy-specific config in Telegraf (`host` and `port`), it resolved their issue. We want to direct future users to that config option.

@keep94 I think we might update the comment in the commit to something like:

`## Url for Wavefront Direct Ingestion. For Wavefront Proxy Ingestion, see the 'host' and 'port' options below.`

Comment (powersj) [reply plugins/outputs/wavefront/README.md:8]: @oppegard thank you for the background, that helps greatly. I agree with your suggestion and would like to see that change made.

Comment (keep94) [reply plugins/outputs/wavefront/README.md]: oppegard's suggestion applied to this PR.

**Pull request merged**